### ISSUE TITLE
bugs out

### ISSUE THREAD

**Issue by alexstaj** [sararselitsky/tDRmapper#4]
```
$ perl Scripts/TdrMappingScripts.pl hg38_mature_pre_for_tdrMapper.fa  input/B1.fastq.gz 
Warning messages:
1: `show_guide` has been deprecated. Please use `show.legend` instead. 
2: Non Lab interpolation is deprecated 
Warning message:
`show_guide` has been deprecated. Please use `show.legend` instead. 
Error in zero_range(from) : x must be length 1 or 2
Calls: ggsave ... train_cartesian -> <Anonymous> -> f -> rescale -> zero_range
Execution halted
```

**Comment by alexstaj** [comment]
it didnt map to any pre so it bugs out at producing the pre-pdf.  what should I do?

**Comment by sararselitsky** [comment]
The whole thing stops because there are no pre-tRNA reads?

**Comment by alexstaj** [comment]
Yes it just stops.  Execution halted it says.  The last file produced is
the pre pdf which is blank.

**Comment by sararselitsky** [comment]
What files are outputted when you ran this?

**Comment by alexstaj** [comment]
so it produced:
B1.fastq.gz.hq_cs.mapped
B1.fastq.gz.hq_cs.mapped.covg.txt
B1.fastq.gz.hq_cs.mapped.covgPre.txt (47 bytes - empty)
B1.fastq.gz.hq_cs.mapped.speciesInfo.txt
B1.fastq.gz.hq_cs.mapped.top50covg.txt
B1.fastq.gz.hq_cs.mapped.top50covgPre.txt (47 bytes - empty)
B1.fastq.gz.hq_cs.mapped.top50covg.txt.top50_tdr.pdf
B1.fastq.gz.hq_cs.mapped.top50covgPre.txt.top50Pre_tdr.pdf (3.8kB - empty)

and in the directory of B1.fastq.gz it produced:

B1.fastq.gz.hq_cs
B1.fastq.gz.hq_cs.notEM
B1.fastq.gz.hq_cs.notEM.notPre
B1.fastq.gz.hq_cs.notEM.notPre.not1MM
B1.fastq.gz.hq_cs.notEM.notPre.not1MM.not1del
B1.fastq.gz.hq_cs.notEM.notPre.not1MM.not1del.not2MM
B1.fastq.gz.hq_cs.notEM.notPre.not1MM.not1del.not2MM.not2del
B1.fastq.gz.hq_cs.notEM.notPre.not1MM.not1del.not2MM.not2del.not3ConsDel

hi I shared a folder to your google drive with my fastq and the hg38 mature and pre fasta.
